src/main/java/com/idukbaduk/itseats/member/controller/MemberController.java

```diff
@@ -28,8 +28,22 @@ public ResponseEntity<BaseResponse> signUp(@RequestBody @Valid CustomerCreateReq
         );
     }
 
+    /**
+     * 현재 로그인한 사용자 정보 조회
+     * 
+     * JWT 토큰을 통해 인증된 사용자의 정보를 반환합니다.
+     * 
+     * @param userDetails Spring Security에서 주입하는 사용자 정보
+     *                    JWT 인증 필터에서 설정된 인증 정보
+     * @return 현재 사용자 정보
+     * @throws RuntimeException 인증되지 않은 사용자인 경우
+     */
     @GetMapping("/me")
     public ResponseEntity<BaseResponse> getCurrentMember(@AuthenticationPrincipal UserDetails userDetails) {
+        // JWT 인증 필터에서 설정된 인증 정보가 없는 경우 처리
+        if (userDetails == null) {
+            throw new RuntimeException("인증되지 않은 사용자입니다.");
```

Comment:
커스텀 예외를 사용하는 것이 좋을 것 같습니다!
```suggestion
            throw new MemberException(MemberErrorCode.UNAUTHORIZED_USER);
```
MemberErrorCode에는 
`UNAUTHORIZED_USER(HttpStatus.UNAUTHORIZED, "인증되지 않은 사용자입니다.");`
추가하시면 됩니다!